README.md

```diff
@@ -8,7 +8,16 @@
 ## Working Changes
 [Read the working changes to the spec](spec.md)
 
-These changes have been accepted by the working group for the next version of the specification, however are still subject to change.
+These changes have been accepted by the working group for the next version of
+the specification, however are still subject to change.
+
+## Alpha Features
+
+We release some features at an alpha level to master before they become an
```

Comment:
To simplify the explanation, I recommend avoiding references to the master branch (because alpha features will be merged into master) or suggesting that alpha features are not part of the spec (IMO they are, just experimental). Suggestion:

> Some enhancements to the API specification are experimental and subject to backward incompatible change or removal. These features are clearly documented as "Alpha." Alpha features allow broker and platform authors implementing the API to test these features and provide feedback before they become stable and subject to our [Change Policy](#change-policy).  Alpha features are documented [here](alpha-features.md).